src/keygenerator.js

```diff
@@ -9,8 +9,6 @@ const publicKey = key.getPublic('hex');
 const privateKey = key.getPrivate('hex');
 
 // Print the keys to the console
-console.log();
-console.log('Your public key (also your wallet address, freely shareable)\n', publicKey);
+console.log('\nYour public key (also your wallet address, freely shareable)\n', publicKey);
 
-console.log();
-console.log('Your private key (keep this secret! To sign transactions)\n', privateKey);
\ No newline at end of file
+console.log('\nYour private key (keep this secret! To sign transactions)\n', privateKey);
```

Comment:
Not sure these changes are necessary. They make the code shorter, but a bit harder to read. Is a bit personal preference I guess.

Reply:
yeah it is, but... i think it makes it more readable, but you can decide if it should be so, or changed back

Reply:
Let's keep it the way it was.

Reply:
reverted with commit be3c54d44622cf93d517f5340f9d4766d0f0ce1e